rtb/piano_di_progetto_ver0.6.0.typ

```diff
@@ -414,4 +431,42 @@ La risorsa che avrebbe dovuto lavorare alla _dashboard_ è stata quindi rialloca
 
 Tuttavia, a causa sia di stime troppo ottimistiche da parte del responsabile che dallo stretto legame tra lo studio delle metriche e la stesura delle Norme di Progetto, si è riscontrato un ritardo nel completare queste attività entro i tempi preventivati. Queste quindi si protrarranno anche nella prima parte prossimo _sprint_.
 
-Per quanto riguarda l'Analisi dei Requisiti l'attività di stesura è proseguita come previsto. Il gruppo ha riscontrato delle difficoltà nell'identificazione dei casi d'uso e, più nello specifico, nel delineare il confine tra analisi dei requisiti e progettazione. È risultato quindi utile il confronto avvenuto con il professor Cardin, che ha permesso di chiarire molti dubbi. Ciò si è ripercosso anche nella stesura della parte riguardante la descrizione dei casi d'uso all'interno delle Norme di Progetto, che ora riporta per iscritto le _best practices_ imparate dal gruppo fino a questo momento.
\ No newline at end of file
+Per quanto riguarda l'Analisi dei Requisiti l'attività di stesura è proseguita come previsto. Il gruppo ha riscontrato delle difficoltà nell'identificazione dei casi d'uso e, più nello specifico, nel delineare il confine tra analisi dei requisiti e progettazione. È risultato quindi utile il confronto avvenuto con il professor Cardin, che ha permesso di chiarire molti dubbi. Ciò si è ripercosso anche nella stesura della parte riguardante la descrizione dei casi d'uso all'interno delle Norme di Progetto, che ora riporta per iscritto le _best practices_ imparate dal gruppo fino a questo momento.
+
+#pagebreak()
+
+=== Sprint 4
+- *Inizio*: 24/12/2024
+- *Fine*: 8/01/2024
+- *Rischi attesi*: #link(<RO-1>)[RO-1], #link(<RO-3>)[RO-3], #link(<RO-4>)[RO-4], #link(<RT-1>)[RT-1]
+
+==== Attività dello sprint
+Le attività che il _team_ si è prefissato di portare a termine durante il quarto _sprint_ sono le seguenti:
+- Redazione del verbale esterno ed interno del 23/12/2024.
+- Codifica dello _script_ per il calcolo automatico dell'_#rifGlossario("indice Gulpease")_.
+- Ampliamento del documento Norme di Progetto, in particolare occupandosi delle sezioni:
+  - Metriche di qualità del prodotto
+  - Metriche di qualità del processo.
+- Redazione completa del Piano di Qualifica con le metriche individuate nel punto sopra.
+- Continuazione del documento di Analisi dei Requisiti, in particolare:
+  - Rifinire e completare i casi d'uso
+  - Individuare e redarre in maniera completa i Requisiti.
+- Creazione del nuovo _#rifGlossario("stream processor") custom_ sfruttando #rifGlossario("Spring"), con il _database_ popolato e l'integrazione con _#rifGlossario("LangChain")_.
```

Comment:
Secondo le NdP LangChain non va in corsivo visto che, più che un termine inglese, è il nome proprio di un software/prodotto